### PR TITLE
Update swg_charter.adoc

### PR DESCRIPTION
**Pull request by FransKnibbe**
replace explicit version identifier

### CODE REVIEW

File: geosparql_2.0_swg_charter/swg_charter.adoc [hunk 1]
```diff
@@ -1,4 +1,4 @@
-:swg-name: GeoSPARQL 2.0 SWG
+:swg-name: GeoSPARQL update SWG
```
Comment (jabhay) [inline geosparql_2.0_swg_charter/swg_charter.adoc:1]: In this instance, I'd just remove it.

```suggestion
:swg-name: GeoSPARQL SWG
```

**Review by jabhay — CHANGES_REQUESTED** [review]

**Comment by FransKnibbe** [comment]
I think "GeoSPARQL SWG" is the name of the group that made GeoSPARQL 1.0. So that could lead to confusion. How about GeoSPARQL II SWG? The roman number indicates a new version without the need for a fixed version identifier.

**Comment by lieberjosh** [comment]
It's perfectly fine for a group that is updating a standard to have the same name, but it might be time to consider whether this new standard will have a substantially different scope than GeoSPARQL, including graph representations and operations that are not strictly covered by OWL and/or SPARQL. A different name may then be appropriate, such as "GeoGraph SWG".

**Comment by FransKnibbe** [comment]
Josh, thank you for clarifying. I guess in my mind I was already seeing two URI's leading to different data :-)
Yes, I agree that development may take us down a path that doesn't entirely match SPARQL any longer. For that matter, I would not be sorry if the "Geo" part will not be entirely applicable any longer when the update is finished. That said, the correctness of naming may not be such an issue if the new product is modularized (more). Then I can imagine some modules not being applicable to SPARQL, or to geography only. But if other parts of the standard still are, then the name GeoSPARQL would still be appropriate.  After all, a maximum amount of compatibility with the current standard is still desirable, isn't it?

**Review by jabhay — APPROVED** [review]
I'll amend this

**Pull request merged**